pkg/contracts/src/CVStrategy.sol

```diff
@@ -755,7 +755,6 @@ contract CVStrategy is BaseStrategy, IPointStrategy, ERC165 {
                 totalStaked -= previousStakedPoints - stakedPoints;
                 proposal.stakedAmount -= previousStakedPoints - stakedPoints;
             }
-            //@todo: should emit event
```

Comment:
the event was emitted?

Reply:
Yeah it was SupportAddedd event you meant right ? in the if under
`if (proposal.blockLast == 0) {
                proposal.blockLast = block.number;
            } else {
                _calculateAndSetConviction(proposal, previousStakedPoints);
                emit SupportAdded(_sender, proposalId, stakedPoints, proposal.stakedAmount, proposal.convictionLast);
            }
`

Reply:
ok, gonna remove